pvnet_app/app.py

```diff
@@ -77,6 +77,12 @@
 model_name_ocf_db = "pvnet_v2"
 use_adjuster = os.getenv("USE_ADJUSTER", "True").lower() == "true"
 
+# If environmental variable is true, the sum-of-GSPs will be computed and saved under a different
+# model name. This can be useful to compare against the summation model and therefore monitor its
+# performance in production
+save_gsp_sum = os.getenv("SAVE_GSP_SUM", "False").lower() == "true"
```

Comment:
i prefer `int(os.getenv("SAVE_GSP_SUM", "False"))`, as I think it covers a few more edge cases

Reply:
you are right

Reply:
yea stick with what you have